docker-compose.yml

```diff
@@ -63,16 +63,14 @@ services:
       - rabbitmq
 
   web_server:
-    image: al-web-server:latest
+    image: quay.io/almalinuxorg/albs:latest
     env_file:
       - vars.env
     ports:
       - "8088:8000"
-    build:
-      dockerfile: Dockerfile
-      context: .
     volumes:
       - "./alws:/code/alws"
+      - "./requirements.txt:/code/requirements.txt"
```

Comment:
we already copy `requirements.txt` in Dockerfile's, maybe we need to remove it from here or from Dockerfile's then?

Reply:
Look at the above comment.